pkg/kubeapiserver/authenticator/config.go

```diff
@@ -150,7 +150,7 @@ func (config AuthenticatorConfig) New() (authenticator.Request, *spec.SecurityDe
 	if config.BootstrapToken {
 		if config.BootstrapTokenAuthenticator != nil {
 			// TODO: This can sometimes be nil because of
```

Comment:
> // TODO: This can sometimes be nil because of

the suspense is killing me

Reply:
This is proprietary comment. If you want to see the rest, you must by my long term support plan.